Sources/TensorFlow/Bindings/RawOpsAugmented.swift

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Augment the `_Raw` interface with ops that take Swift integers for the
+// shape attributes rather than requiring that they be passed as `Int32` tensors.
+// This is useful for implementations that do not require the parameters be
+// passed as Tensors.
+extension _Raw {
+  public static func argMax<
```

Comment:
Please use `argmax` for consistency.
```suggestion
  public static func argmax<
```

Reply:
These names are chosen to match the tf op names (be overloads). I'd like to keep them matching the raw ops they overload.

Reply:
This name is chosen to overload the existing raw ops generated name.